src/Compilers/CSharp/Portable/BoundTree/NullabilityRewriter.cs

```diff
@@ -71,7 +71,7 @@ private BoundNode VisitBinaryOperatorBase(BoundBinaryOperatorBase binaryOperator
             while (stack.Count > 0);
 
             Debug.Assert(currentBinary != null);
-            return currentBinary;
+            return currentBinary!;
```

Comment:
Is this part of the change?

Reply:
Yes.  Apparently the version of `Debug.Assert` used here is not annotated.

---
In reply to: [303279357](https://github.com/dotnet/roslyn/pull/34916#discussion_r303279357) [](ancestors = 303279357)